x/market/keeper/grpc_query_test.go

```diff
@@ -141,15 +141,15 @@ func TestGRPCQueryOrders(t *testing.T) {
 				req = &types.QueryOrdersRequest{
 					Filters: types.OrderFilters{
 						OSeq:  37,
-						State: types.Order_State(8),
+						State: "matched",
```

Comment:
Is this generated or could we set/reference this to be a const somewhere? (plus subsequent literal strings below)

Reply:
/cc @akhilkumarpilli

Reply:
Actually, I am sending value directly as `State` is string type. I will change it to `types.OrderMatched.String()` so it will be clear. I will update other strings too.